_layouts/state-page.html

```diff
@@ -349,11 +350,9 @@
         {% assign units = product[1].units | downcase %}
 
         <h4>{{ product[1].name }} ({{ units | default: 'units' }})</h4>
-        <p>{{ state_name }} produced
-          {% if volume %}{{ volume | intcomma }}
+        <p>{% if volume %}{{ volume | intcomma }}
           <span class="units">{{ units | default: 'units' }}</span>
-          of {% else %}no{% endif %}
-          {{ product[1].name | downcase }} in {{ year }}. Historically:</p>
+          of {{ product[1].name | downcase }} were {% else %}No {{ product[1].name | downcase }} was {% endif %} produced on federal land in {{ state_name }} in {{ year }}.</p>
```

Comment:
Similarly, can this be broken up onto multiple lines? If it creates weird spacing we can leave it, but otherwise, something like this might keep things _slightly_ more legible

```
<p>
  {% if volume %}
    {{ volume | intcomma }}
    <span class="units">{{ units | default: 'units' }}</span>
    of {{ product[1].name | downcase }} were 
  {% else %}
    No {{ product[1].name | downcase }} was 
  {% endif %} 
  produced on federal land in {{ state_name }} in {{ year }}.
</p>
```